### PR TITLE
Remove broken link (and "Distributed Systems" section)

### PR DESCRIPTION
**Pull request by gpoussel**
Typhoon project link is dead.

### CODE REVIEW

**Comment by fogfish** [comment]
Too bad Zalando dropped my project from they open source repository... Let's think what we can do about it.

**Comment by llelf** [comment]
@fogfish Replace it with your repo? :)
and https://web.archive.org/web/20170302004928/https://github.com/zalando/typhoon

**Comment by fogfish** [comment]
Yeah, indeed! 
https://github.com/fogfish/typhoon

I am currently preparing a new release to it, I'd like to simplify it a bit and fit to use-case of small company.

File: README.md [hunk 1]
```diff
@@ -122,11 +122,6 @@ A curated list of amazingly awesome Erlang libraries, resources and shiny thing
 
 * [docker-erlang](https://github.com/synlay/docker-erlang) - Basic Docker Container Images for Erlang/OTP.
```
Comment (drobakowski) [inline README.md]:
```suggestion

## Distributed Systems
 *Tools for stress/load testing, latency issues, etc. across microservices.*

 * [Typhoon](https://github.com/fogfish/typhoon) - Stress and load testing tool for distributed systems that simulates traffic from a test cluster toward a system-under-test (SUT) and visualizes related latencies.
```

**Comment by gpoussel** [comment]
Of course, ok!

**Comment by drobakowski** [comment]
Great, thanks @gpoussel!

**Pull request merged**